### ISSUE TITLE
Triple screen

### ISSUE THREAD

**Issue by ctbr63** [mika-n/NGPCarMenu#3]
Hi
Version 1.02 works fine with my triple screen setup.
There is only one small isse. The text about 3D car models and author is shown in the list of cars, see attached screenshot.

One idea for improvement, I would be nice if the plugin stored the skin that was used to create the preview and if the skin have been changed created a new preview.

Thankyou for creating this plugin!

![NGPCarMenu_triple_screen](https://user-images.githubusercontent.com/65561837/82244229-350a1a80-9941-11ea-9fba-8c3ada332c87.jpg)

**Comment by mika-n** [comment]
Aah. Okey. The placement of the "3D Model" info box indeed is a clumsy calculation at the moment. The coordinate system of RBR is a bit weird because the game is internally 640x480. FixUp plugin aspect ratio fix may also add a twist here. Therefore coordinates of in-game objects don't follow the native "pixel by pixel" coordinates (those follow DX9 "world coordinates"). I could not quite figure out how to properly re-map in-game coordinates to a real 2D screen coordinates or vice-versa. 

As a quick-and-dirty solution I will add INI property to tweak the placement of 3D model info box. Then there is at least a self-service workaround if the existing "automagical" coordinate re-map calculation fails to do the job until better re-map formula is found.

The next version will have this "self-service bug fix" feature, so keep eye on the releases.

**Comment by mika-n** [comment]
> One idea for improvement, I would be nice if the plugin stored the skin that was used to create the preview and if the skin have been changed created a new preview.

Indeed. Nice idea. However, I have to think about and test are there any potential problems if the new preview is re-generated automatically when the game is launched or "Select Car" menu shown. The replay logic does make it a bit problematic because it takes back to main menu... Let's see how it goes.....  Minimun would be to show a notification text if there are changes and a preview is missing.

**Comment by mika-n** [comment]
V1.03 fixes (workarounds) the "3D model into textbox" location issue. 

The default location is now a bit better (may not be perfect in all resolutions), but a new **Car3DModelInfoPosition** ini file entry makes it possible to tweak the location (it can be placed even on top of the car preview image if someone wants to place it at, for example, bottom left corner).
